granary/tests/test_reddit.py

```diff
@@ -0,0 +1,123 @@
+from oauth_dropins.webutil import testutil
+from oauth_dropins.webutil import util
+
+from granary import reddit
+
+import copy
+import json
+
+class FakeRedditor():
+  id = '59ucsixw'
+  name = 'bonkerfield'
+  subreddit = {'display_name': 'u_bonkerfield',
+   'title': '',
+   'icon_img': 'https://styles.redditmedia.com/t5_2az095/styles/profileIcon_ek6onop1xbf41.png?width=256&height=256&crop=256:256,smart&s=81fbf5a84c69533b5679f46638353d9e8471b817',
```

Comment:
minor, pls remove the `s` query param? may be innocuous but looks like a token. same elsewhere